src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -382,6 +382,7 @@ INTEGRATED_CHANNELS_API_CHUNK_TRANSMISSION_LIMIT:
 JWT_EXPIRATION: 30
 LANGUAGE_CODE: en
 LANGUAGE_COOKIE: {{ env "ENVIRONMENT" }}-openedx-language-preference
+LEARN_AI_API_URL: {{ key "edxapp/learn-ai-domain" }}  # Added for ol_openedx_chat
```

Comment:
@blarghmatey Consider this a dummy value, It should point to ``<LEARN_API_URL>/http/recommendation_agent/`, I don't know the variable name to use here.

This should be as per https://github.com/mitodl/ol-infrastructure/issues/2981.